2017/atuttle7.txt

```diff
@@ -0,0 +1,187 @@
+
+                                   A
+                                   M
+                                   M
+                                   M
+                                   M
+                                   M
+                                   M
+                                   M
+                                  /M\
+                                 '[V]'
+                                  [A]
+                                 [,-']
+                                 [/"\]
+                                 / _ \
+                                / / | \
+                               / /_O_| \
+                              /______|__\
+                              |=_==_==_=|
+                              |  |   |  |
+                             V|  |.V.|__|V
+                             A|  |'A'| =|A
+                              |__|___|= |
+                              |__|___| =|
+                              |####|####|
+                             |    o|     |
+                             |     |     |
+                             |     |     |
+                            |      |      |
+                            |      |      |
```

Comment:
This is going to get cut off here. Just FYI

I would recommend shrinking it a lot